resources_pages/installation_instructions.md

```diff
@@ -1,11 +1,31 @@
 ---
 layout: page
-title: Install Guide
+title: Installation Guide
 subtitle: For the MDS software stack
 ---
 
-Below are links to the software installation guide used for the MDS program for the three operating systems we support. We provide these installation instructions to our students on the first day of the program. The following people contributed to these instructions: Anmol Jawandha, Tomas Beuzen, Rodolfo Lourenzutti, Joel Ostblom & Tiffany Timbers.
+These instructions will walk you through how to install the required Data Science software stack for the UBC Master of Data Science program. Before starting with the installation instructions, ensure that your laptop meets our program requirements. **Students' whose laptops do not meet the requirements specified above will not be able to receive technical assistance from the MDS team in troubleshooting installation issues.**
+
+## Laptop requirements
+
+ - Runs one of the following operating systems: macOS Big Sur (10.4.x or 10.5.x), Ubuntu 20.04, Windows 10 Professional, Enterprise or Education; version 2004.
```

Comment:
Just saw a typo in the MacOS version number, which should be 11.4.x or 11.5.x.

Reply:
Good catch, thank you!